docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
+= Adding a volume component to a devfile
+
+[role="_abstract"]
+You can use a volume component to share files among container components and collaborate easily with other teams during the development process.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-components-to-a-devfile.adoc[]
+
+.Procedure
+
+* Define a `volume` component in your devfile.
++
+.A minimal `volume` component
```

Comment:
A minimal volume component does not need to specify the size attribute.   In this  case, the default size is determined by the tools implementing the spec:
```
- name: mydevfile
  volume: {}
```

If we want to add other examples, we can include the one where size is specified but again, it's tools specific on how it's handled.   

Size Example:
`Specify a volume size on creation or update the size of an existing volume.`  

```
  - name: mydevfile
    volume:
      size: 2Gi
```

`Note: Implementation is entirely dependent on the consuming tools and may be subject to their limitations.  Refer to your tools documentation for further details`

Ephemeral Example:
`Store ephemeral volumes persistently across restarts`

```
  - name: mydevfile
    volume:
      ephemeral: true
      size: 200G
```

Reply:
Thanks, I applied the feedback and let me know what you think!